force-app/main/default/classes/SOQL.cls

```diff
@@ -50,9 +54,16 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Id> contactsByLastName () {
+        String lastName1 = 'Smith';
+        String lastName2 = 'Jones';
+        List<Id> contactIds = new List<Id>();
+        List<Contact> contactsLastName = [SELECT Name, Id FROM Contact WHERE LastName = :lastName1 OR LastName = :lastName2];
```

Comment:
I would use the IN operator and a Set of names to compare against rather than checking for each one individually.